guidelines/index.html

```diff
@@ -1170,6 +1177,15 @@
 					<p>audio description that is added to an audiovisual presentation by pausing the <a>video</a> so that there is time to add additional description</p>
 					<p>This technique is only used when the sense of the <a>video</a> would be lost without the additional <a>audio description</a> and the pauses between dialogue/narration are too short.</p>
 				</dd>
+        <dl>
+		          <dt><dfn>easily available (or easily available mode or setting) one or more of the following are true:</dfn></dt>
+                  <dd>
+        <ul>
+  <li>can be set one time with as wide a scope as possible (such as using the standards of the OS, From ISO 9241-112 or GPII when available) </li>
+  <li>has the option to save or change the setting, where available interoperably, but also for the scope of the set of web pages </li>
```

Comment:
The latter part of this requirement is dangling (i.e. "but also..." only contains a prepositional phrase).  I'd like to help with rewording, but I'm confused by the intent of this bullet versus the previous one.  They both seem to be dealing with scope of the setting.  Can you clarify the distinction?